src/oop/app/src/Transporters/GuzzleAdapter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace src\oop\app\src\Transporters;
+
+/**
+ * Class GuzzleAdapter use GuzzleHttp Client for getting the page content.
+ * Adopts site for parcing.
+ * Note: About GuzzleHttp Client you can read here:
+ * https://docs.guzzlephp.org/en/stable/
+ */
+class GuzzleAdapter implements TransportInterface
+{
+    private \GuzzleHttp\Client $client;
+
+    public function __construct()
+    {
+        $this->client = new \GuzzleHttp\Client(['base_uri' => 'http://httpbin.org/','timeout'  => 2.0]);
```

Comment:
please pass Client as dependency (do not use new Client())

Reply:
Done